packages/editor/src/lib/editor/Editor.ts

```diff
@@ -710,8 +711,10 @@ export class Editor extends EventEmitter<TLEventMap> {
 			this._tickManager.start()
 		})
 
-		this.licenseManager = new LicenseManager(this)
-		this.licenseManager.getLicenseFromKey(licenseKey)
+		this.licenseManager = new LicenseManager()
+		const license = this.licenseManager.getLicenseFromKey(licenseKey)
+		this.watermarkManager = new WatermarkManager(this)
```

Comment:
A bit unsure if we need these managers?

I can see us using the license manager in case we wanted it to hold the license information for displaying it in some place. Not sure about the watermark manager though. Still feels better than having it inside the license manager though.

Reply:
It was originally a React function component, I changed it based on Steve's suggestion that it should be created on mount. It doesn't need to be a class, or be called a manager, I just couldn't come up with another way of structuring it.

Reply:
Yeah, same here, don't have a better idea. I mean, we could just have a function since it doesn't really store any state atm.